pylintrc

```diff
@@ -24,7 +29,10 @@ disable=too-many-locals,
         global-statement,
         too-many-branches,
         global-variable-not-assigned,
-        redefined-argument-from-local
+        redefined-argument-from-local,
+        line-too-long,
+        too-many-return-statements,
+        unbalanced-tuple-unpacking
```

Comment:
My thoughts:
- `line-too-long`: One of the biggest stylistic advantages of linting. Strong vote to keep.
- `too-many-return-statements`: I'm fine with simple functions with many return statements. That said, we only need to exclusion once in [`FusedScaleMaskSoftmax.is_kernel_available`](https://github.com/NVIDIA/TransformerEngine/blob/3b813689c16fd5fd286bd2302fc8242afb62b44f/transformer_engine/pytorch/softmax.py#L339).
- `unbalanced-tuple-unpacking`: I don't see a reason to exclude it and it helps with safety.
```suggestion
        redefined-argument-from-local
```

Reply:
Line length will be handled by the formatter, not the linter.